packages/ui/src/app/pages/Proposals/ProposalPreview.tsx

```diff
@@ -158,7 +158,7 @@ export const ProposalPreview = () => {
             </BadgeAndTime>
           </RowGapBlock>
 
-          {(proposal.status === 'dormant' || votingRounds.length > 1) && (
+          {(proposal.status === 'dormant' || votingRounds.length >= 1) && (
```

Comment:
This is not the right way.

Now the tabs should be shown for any proposal with a `constitutionality > 1`  so `proposal.status` and `votingRounds.length` is irrelevant.